common/environment/setup/misc.sh

```diff
@@ -18,5 +18,6 @@ GNU_SITE="https://ftp.gnu.org/gnu"
 FREEDESKTOP_SITE="https://freedesktop.org/software"
 KDE_SITE="https://download.kde.org/stable"
 VIDEOLAN_SITE="https://download.videolan.org/pub/videolan"
+XFCE_SITE="https://archive.xfce.org"
```

Comment:
Drop the commit with this change, we agreed to not add this variable in another PR.

Reply:
done

Reply:
You also can't use this variable in the package when it's not defined.